src/Enhavo/Bundle/SidebarBundle/Resources/config/routing/sidebar.yml

```diff
@@ -0,0 +1,111 @@
+enhavo_sidebar_sidebar_index:
+    options:
+        expose: true
+    path: /sidebar/sidebar/index
+    methods: [GET]
+    defaults:
+        _controller: enhavo_sidebar.controller.sidebar:indexAction
+        _sylius:
+            viewer:
+                type: sidebar
+                title: sidebar.label.sidebar
+                translationDomain: EnhavoSidebarBundle
+
+enhavo_sidebar_sidebar_create:
+    options:
+        expose: true
+    path: /sidebar/sidebar/create
+    methods: [GET,POST]
+    defaults:
+        _controller: enhavo_sidebar.controller.sidebar:createAction
+        _sylius:
+            redirect: enhavo_sidebar_sidebar_update
+            sortable: true
```

Comment:
please remote this `sortable`, `sortable_position` and `sorting_strategy` configs, because they are not used in enhavo 0.7 anymore